### DISCUSSION TITLE
Virtualbox broken after upgrading to 6.3.3-surface

### DISCUSSION THREAD

**Discussion by RichardODonoghue** [linux-surface/linux-surface#1154] [Q&A]
After updating to 6.3.3-surface I found that I was no longer able to start up VMS. 

I tried uninstalling and reinstalling VB I got this error: 
```
ERROR (dkms apport): kernel package linux-headers-6.3.3-surface is not supported
Error! Bad return status for module build on kernel: 6.3.3-surface (x86_64)
```

![image](https://github.com/linux-surface/linux-surface/assets/1446236/b74f74db-f7c5-4bc0-bd5b-0be6e6ef3424)

Anyone have any solutions other than downgrading?

**Comment by qzed** [comment]
Unfortunately that happens occasionally. The problem is that Ubuntu uses older kernels, so any dkms packages that they ship are designed for those. Whereas we generally try to keep our kernels on the latest release (since both Arch and Fedora do that). When newer kernels change some API, those modules can break and may no longer compile. You can try to either get a newer version from somewhere (e.g. upstream sources of the packages) or, as you've already mentioned, downgrade.

I am still hoping that at some point we can get "older" kernels to mostly work as well, but backporting things isn't always easy. For example, I'm trying to keep 6.1 (the latest LTS) up-to-date, but cameras may already not fully work on that. So all in all, that's a bit difficult sometimes.

Depending on the device and on the kernel that Ubuntu 23.04 ships with, you may already get away with the default kernel for most things (but expect touch and cameras to not work).
